lib/chirp-workflow/core/src/db/mod.rs

```diff
@@ -101,15 +117,15 @@ pub struct ActivityEventRow {
 	pub location: Vec<i64>,
 	pub name: String,
 	pub input_hash: Vec<u8>,
-	pub output: Option<String>,
+	pub output: Option<serde_json::Value>,
```

Comment:
this allocates a bunch of memory. opt to use rawvalue (https://docs.rs/sqlx/latest/sqlx/types/struct.JsonRawValue.html#example)

Reply:
You can only read from sqlx into a `&serde_json::value::RawValue`. Adding lifetimes will likely make all of our types a mess and isn't worth the gain. Funny enough if we went back to deserializing into (but not storing as) strings that would probably be the best of both worlds

Reply:
will leave as value for now, not worth the time